BitwardenShared/UI/Auth/Login/SingleSignOn/SingleSignOnProcessor.swift

```diff
@@ -149,6 +148,8 @@ final class SingleSignOnProcessor: StateProcessor<SingleSignOnState, SingleSignO
                 await handleLoginTapped()
             }
```

Comment:
shouldn't we add a else case here and default to
```state.identifierText = services.stateService.rememberedOrgIdentifier ?? ""```
?

Reply:
👍  I agree, also it could be transformed into a guard so to have less code blocks nested.

Reply:
Good catch, updated!